packages/@aws-cdk/aws-apigateway/lib/domain-name.ts

```diff
@@ -39,6 +39,13 @@ export interface DomainNameOptions {
    * @default SecurityPolicy.TLS_1_0
    */
   readonly securityPolicy?: SecurityPolicy
+
+  /**
+   * The mutual TLS authentication configuration for a custom domain name.
+   * @default - mTLS is not configured.
+   */
+  readonly mutualTlsAuthentication?: MutualTlsAuthenticationAttributes
```

Comment:
mTLS is the industry standard abbreviation. Let's use that.

```suggestion
  readonly mtls?: MTLSAttributes
```